pom.xml

```diff
@@ -282,9 +282,9 @@
           </jvmArgs>
           <javacArgs>
             <javacArg>-source</javacArg>
-            <javacArg>${java.version}</javacArg>
+            <javacArg>1.7</javacArg>
```

Comment:
we require jdk8 because of the transitive dependency on htsjdk through Hadoop-BAM